copyright.txt

```diff
@@ -1,9 +0,0 @@
-﻿Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
```

Comment:
Did you accidentally delete this file?

Reply:
not that im aware of, let me ad it back in